components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/internal/service/impl/password/PasswordRecoveryManagerImpl.java

```diff
@@ -461,14 +573,15 @@ private PasswordResetCodeDTO buildPasswordResetCodeDTO(String resetCode) {
      * @return PasswordRecoverDTO object
      */
     private PasswordRecoverDTO buildPasswordRecoveryResponseDTO(String notificationChannel, String confirmationCode,
-                                                                String resendCode) {
+                                                                String resendCode, String recoveryFlowId) {
```

Comment:
update the method comment

Reply:
Fixed with ab489ffd47d3bb04acd840bdff99979c579d20aa